### ISSUE TITLE
Building beautiful UIs with Flutter - Page 7 - Broken link

### ISSUE THREAD

**Issue by racouster** [flutter/flutter#100002]
The link on the text CrossAxisAlignment.start goes to https://api.flutter.dev/flutter/rendering/CrossAxisAlignment-class.html
which has content:
Page Not Found...

**Comment by bryancalisto** [comment]
True. The link should be [https://api.flutter.dev/flutter/rendering/CrossAxisAlignment.html#start](https://api.flutter.dev/flutter/rendering/CrossAxisAlignment.html#start)
